streampark-flink/streampark-flink-submit/streampark-flink-submit-core/src/main/scala/org/apache/streampark/flink/submit/impl/YarnPerJobSubmit.scala

```diff
@@ -101,14 +101,15 @@ object YarnPerJobSubmit extends YarnSubmitTrait {
 
       }
       val applicationId = clusterClient.getClusterId
+      val jobManagerUrl = clusterClient.getWebInterfaceURL
       logInfo(
         s"""
            |-------------------------<<applicationId>>------------------------
            |Flink Job Started: applicationId: $applicationId
            |__________________________________________________________________
            |""".stripMargin)
 
-      SubmitResponse(applicationId.toString, flinkConfig.toMap)
+      SubmitResponse(applicationId.toString, flinkConfig.toMap, "", jobManagerUrl)
```

Comment:
you can improve this code: SubmitResponse(applicationId.toString, flinkConfig.toMap, jobManagerUrl = jobManagerUrl);